### PR TITLE
lib/runtime: fix clear_prefix w/ TestRuntimeStorage

### PR DESCRIPTION
**Pull request by noot**
## Changes

- fix clear_prefix w/ TestRuntimeStorage
- update polkadot execute_block test to load wasm from `:code` genesis storage (test overall still needs work)

## Tests

```
go test ./lib/runtime/wasmer
```

## Checklist

- [x] I have read [CODE_OF_CONDUCT](https://github.com/ChainSafe/gossamer/blob/development/.github/CODE_OF_CONDUCT.md) and [CONTRIBUTING](https://github.com/ChainSafe/gossamer/blob/development/.github/CONTRIBUTING.md) 
- [x] I have provided as much information as possible and necessary
- [x] I have reviewed my own pull request before requesting a review
- [ ] All integration tests and required coverage checks are passing

## Issues

- related to #1105

### CODE REVIEW

**Review by RyRy79261 — APPROVED** [review]

**Review by RyRy79261 — COMMENTED** [review]
image
Test fails on my machine

**Comment by noot** [comment]
@RyRy79261 that file is in the code, was it deleted somehow? can you do `git checkout development chain/polkadot/genesis-raw.json` and retry

**Review by RyRy79261 — APPROVED** [review]
Was minor issue local, tests passing

File: lib/runtime/test_helpers.go [hunk 1]
```diff
@@ -270,6 +271,21 @@ func (trs *TestRuntimeStorage) GetChild(keyToChild []byte) (*trie.Trie, error) {
 // ClearPrefix ...
 func (trs *TestRuntimeStorage) ClearPrefix(prefix []byte) {
 	trs.trie.ClearPrefix(prefix)
+
+	iter := trs.db.NewIterator()
+
+	for iter.Next() {
```
Comment (arijitAD) [inline lib/runtime/test_helpers.go:277]: I think you can seek to a particular key or prefix. https://godoc.org/github.com/dgraph-io/badger#Iterator.Seek

**Review by arijitAD — APPROVED** [review]
Test passed for me as well.

File: lib/runtime/test_helpers.go (hunk 1 above)
Comment (noot) [reply lib/runtime/test_helpers.go:277]: good point, I'll open an issue for this on chainDB to add it to the iterator interface

**Pull request merged**